### DISCUSSION TITLE
Windows 24H2?

### DISCUSSION THREAD

**Discussion by Treachery93** [Open-Shell/Open-Shell-Menu#2016] [General]
Does Openshell still work on Windows 11 24H2?
Openshell v4.4.191 runs fine on Windows 11 23H2 for me.

**Comment by Hadden89** [comment]
It should work fine because not big changes were made to taskbar/start.
But unless you really need some features of 24H2 you can stay with 23H2 as long is supported

**Comment by edersilvago** [comment]
Look
![Screenshot_1](https://github.com/user-attachments/assets/61163f60-5189-4b19-8ea3-a2ccbd8e5c97)
